### ISSUE TITLE
请问如何创建自定义横轴的可视化图像？

### ISSUE THREAD

**Issue by kawabata-tomoko** [SwanHubX/SwanLab-Docs#232]
Swanlab团队你好，在使用Huggingface Trainer（设置report_to="swanlab")后log记录的key包括eval/loss、eval/runtime、eval/samples_per_second、eval/steps_per_second、train/epoch、train/global_step、train/grad_norm、train/learning_rate、train/loss等，但是在网页端可视化训练历程的时候不论是已有的折线图或创建新的折线图，横轴默认都是step且无法更改。无法直接在网页端设置为任意两个key之间的折线图/散点图查看不同指标之间的关系吗？
例如我想令横轴为epoch，纵轴为其他数据以将不同规模的数据集/不同步长训练/不同batchsize训练的时候进行对齐查看在训练的不同阶段上的指标变化，类似

<img width="869" height="622" alt="Image" src="https://github.com/user-attachments/assets/5164da4d-cd6e-4749-82e7-5653eb471271" />
应该如何实现呢？

**Comment by kawabata-tomoko** [comment]
找到相关讨论 https://github.com/SwanHubX/SwanLab/issues/1144#issue-3203041464，关闭此issue
